zuul-core/src/main/java/com/netflix/netty/common/proxyprotocol/HAProxyMessageChannelHandler.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ *      Licensed under the Apache License, Version 2.0 (the "License");
+ *      you may not use this file except in compliance with the License.
+ *      You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *      Unless required by applicable law or agreed to in writing, software
+ *      distributed under the License is distributed on an "AS IS" BASIS,
+ *      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *      See the License for the specific language governing permissions and
+ *      limitations under the License.
+ */
+
+package com.netflix.netty.common.proxyprotocol;
+
+import com.google.common.net.InetAddresses;
+import com.netflix.netty.common.SourceAddressChannelHandler;
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelFutureListener;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelInboundHandlerAdapter;
+import io.netty.handler.codec.haproxy.HAProxyMessage;
+import io.netty.handler.codec.haproxy.HAProxyProtocolVersion;
+import io.netty.util.AttributeKey;
+import io.netty.util.ReferenceCounted;
+import java.net.InetSocketAddress;
+import java.net.SocketAddress;
+
+/**
+ * Copies any decoded HAProxyMessage into the channel attributes, and doesn't pass it any further along the pipeline.
+ * Use in conjunction with HAProxyMessageDecoder if proxy protocol is enabled on the ELB.
+ */
+
+public final class HAProxyMessageChannelHandler extends ChannelInboundHandlerAdapter {
+
+    public static final AttributeKey<HAProxyMessage> ATTR_HAPROXY_MESSAGE = AttributeKey
```

Comment:
Line wrapping should happen at the highest semantic level.  In this case, you should break at the equals `=` sign.   

Here and below.

Reply:
Yep, good catch. Didn't see the auto-formatter apply this.